scripts/demo-projects.android.sh

```diff
@@ -20,8 +20,7 @@ popd
 # as it runs tests in parallel. Also, it installs test-butler on them -
 # even for forward usage.
 pushd examples/demo-react-native-jest
-run_f "npm run test:android-release-ci"
-DETOX_EXPOSE_GLOBALS=0 run_f "npm run test:android-release-ci"
```

Comment:
What is DETOX_EXPOSE_GLOBALS and why is it removed here?

Reply:
Good question, initially I thought we are running the same tests twice without any good reason. However, while revisiting this, looks like the original intention was to test this API: https://wix.github.io/Detox/docs/next/config/behavior/#behaviorinitexposeglobals-boolean

Reply:
I'll revert this and add a comment, great catch.